mds/apis/prv_api/urls.py

```diff
@@ -19,6 +20,9 @@ def get_prv_router():
     prv_router.register(r"service_areas", service_areas.AreaViewSet)
     prv_router.register(r"polygons", service_areas.PolygonViewSet)
     prv_router.register(r"vehicles", vehicles.DeviceViewSet, basename="device")
+    prv_router.register(
+        r"provider_api", provider_api.ProviderApiViewSet, basename="provider_api"
```

Comment:
`basename` may not be required, as there is some defaults, but this is all magic to me.